lz4json on FreeBSD: liblz4 IS installed but lz4.h is reportedly not found

```text
# date ; uname -v
Sat 14 Oct 2017 15:07:48 BST
FreeBSD 12.0-CURRENT #0 r320869: Mon Jul 10 13:57:55 UTC 2017     root@releng3.nyi.freebsd.org:/usr/obj/usr/src/sys/GENERIC 
# pwd
/usr/home/grahamperrin/Documents/com/github/andikleen/lz4json/lz4json-master
# make
cc -g -O2 -Wall  lz4jsoncat.c -llz4 -o lz4jsoncat
lz4jsoncat.c:31:10: fatal error: 'lz4.h' file not found
#include "lz4.h"
         ^~~~~~~
1 error generated.
*** Error code 1

Stop.
make: stopped in /usr/home/grahamperrin/Documents/com/github/andikleen/lz4json/lz4json-master
# pkg info liblz4 | grep -i version
Version        : 1.8.0,1
# 
```

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,4 @@
-CFLAGS := -g -O2 -Wall
-LDLIBS := -llz4
+CFLAGS := -g -O2 -Wall $(shell pkg-config --cflags --libs liblz4)
 
 lz4jsoncat: lz4jsoncat.c
 

